compose/material3/material3/src/desktopTest/kotlin/androidx/compose/material3/KotlinxDatetimeCalendarModelTest.kt

```diff
@@ -0,0 +1,199 @@
+        assertThat(date.year).isEqualTo(2022)
+        assertThat(date.month).isEqualTo(1)
+        assertThat(date.dayOfMonth).isEqualTo(1)
+        // Check that the milliseconds represent the start of the day.
+        assertThat(date.utcTimeMillis).isEqualTo(January2022Millis)
+    }
+
+    @Test
+    fun dateRestore() {
+        val date =
+            CalendarDate(
+                year = 2022,
+                month = 1,
+                dayOfMonth = 1,
+                utcTimeMillis = January2022Millis
+            )
+        assertThat(model.getCanonicalDate(date.utcTimeMillis)).isEqualTo(date)
+    }
+
+    @Test
+    fun monthCreation() {
+        val date =
+            CalendarDate(
+                year = 2022,
+                month = 1,
+                dayOfMonth = 1,
+                utcTimeMillis = January2022Millis
+            )
+        val monthFromDate = model.getMonth(date)
+        val monthFromMilli = model.getMonth(January2022Millis)
+        val monthFromYearMonth = model.getMonth(year = 2022, month = 1)
+        assertThat(monthFromDate).isEqualTo(monthFromMilli)
+        assertThat(monthFromDate).isEqualTo(monthFromYearMonth)
+    }
+
+    @Test
+    fun monthCreation_withRounding() {
+        val date =
+            CalendarDate(
+                year = 2022,
+                month = 1,
+                dayOfMonth = 1,
+                utcTimeMillis = January2022Millis
+            )
+        val monthFromDate = model.getMonth(date)
+        val monthFromMilli = model.getMonth(January2022Millis + 10000)
+        assertThat(monthFromDate).isEqualTo(monthFromMilli)
+    }
+
+    @Test
+    fun monthRestore() {
+        val month = model.getMonth(year = 1999, month = 12)
+        assertThat(model.getMonth(month.startUtcTimeMillis)).isEqualTo(month)
+    }
+
+    @Test
+    fun plusMinusMonth() {
+        val month = model.getMonth(January2022Millis) // 1/1/2022
+        val expectedNextMonth = model.getMonth(month.endUtcTimeMillis + 1) // 2/1/2022
+        val plusMonth = model.plusMonths(from = month, addedMonthsCount = 1)
+        assertThat(plusMonth).isEqualTo(expectedNextMonth)
+        assertThat(model.minusMonths(from = plusMonth, subtractedMonthsCount = 1)).isEqualTo(month)
+    }
+
+    @Test
+    fun parseDate() {
+        val expectedDate =
+            CalendarDate(
+                year = 2022,
+                month = 1,
+                dayOfMonth = 1,
+                utcTimeMillis = January2022Millis
+            )
+        val parsedDate = model.parse("1/1/2022", "M/d/yyyy")
+        assertThat(parsedDate).isEqualTo(expectedDate)
+    }
+
+//    @Test
+//    fun formatDate() {
+//        val date =
+//            CalendarDate(
+//                year = 2022,
+//                month = 1,
+//                dayOfMonth = 1,
+//                utcTimeMillis = androidx.compose.material3.January2022Millis
+//            )
```

Comment:
Can you please add comments that these tests copied from Android sourceSet. And describe a little bit, why they should be commented.

Reply:
Commented tests should be moved to `PlatformDateFormatTest` for each platform

Reply:
> Commented tests should be moved to PlatformDateFormatTest for each platform

Is it because they will have different asserts?  
Currently it's in desktopTest, do they pass here? 

___
a note:

import com.google.common.truth.Truth.assertThat
import org.junit.Test

These won't be supported in non-jvm tests.
`org.junit.Test` can be replaced by `kotlin.test.Test`
but no easy substitution for `com.google.common.truth.Truth.assertThat`, so kotlin.test asserts is the only option - https://kotlinlang.org/api/latest/kotlin.test/kotlin.test/

Reply:
> Is it because they will have different asserts?

I have noticed that some skeletons localizations aren't the same on js and uikit. You can see it on the screenshot. This will probably require different asserts

Reply:
> Currently it's in desktopTest, do they pass here?

They don't, because looks like there is no alternative to date skeletons on jvm.

Reply:
Implemented all tests in `skikoTest`. Skeleton tests are muted for desktop (scince